randovania/gui/widgets/item_tracker_widget.py

```diff
@@ -102,9 +114,25 @@ def __init__(self, tracker_config: dict):
             for resource, label in zip(resources, labels):
                 label.setToolTip(resource.long_name)
 
-            self.tracker_elements.append(Element(labels, resources, text_template, minimum_to_check, field_to_check))
+            self.tracker_elements.append(Element(
+                labels,
+                resources,
+                text_template,
+                minimum_to_check,
+                field_to_check,
+                disabled_image
+            ))
+            if disabled_image is not None:
+                labels.append(disabled_image)
             for label in labels:
-                self._layout.addWidget(label, element["row"], element["column"], row_span, col_span)
+                self._layout.addWidget(
+                    label,
+                    element["row"],
+                    element["column"],
+                    row_span,
+                    col_span,
+                    QtCore.Qt.AlignmentFlag.AlignCenter
```

Comment:
i didn't end up using it in the dread layout, but this change means that if an image has a `col_span` greater than 1, it'll end up centered horizontally instead of aligned left. makes no real difference for `col_span` of 1